ffi/npm/Makefile

```diff
@@ -20,7 +20,7 @@ lib/index.mjs:
 	rustup component add rust-src --toolchain ${PINNED_CI_NIGHTLY}
 	RUSTFLAGS="-Cpanic=abort -Copt-level=s -C link-args=-zstack-size=${WASM_STACK_SIZE} -Clto -Cembed-bitcode" \
 	cargo +${PINNED_CI_NIGHTLY} rustc \
-		-Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort \
```

Comment:
question: why remove panic_abort here and below?

Reply:
It's not a valid argument: https://doc.rust-lang.org/cargo/reference/unstable.html#build-std:~:text=The%20value%20here%20is%20a%20comma%2Dseparated%20list%20of%20standard%20library%20crates%20to%20build.

Reply:
panic_abort certainly is a valid argument. It is one of the crates that is part of the standard library. I guess it works without by accident when panic_immediate_abort is being used, but when it isn't used `-Cpanic=abort` requires `panic_abort` to be used in the `-Zbuild-std` list.

Reply:
Thanks, this isn't documented anywhere